src/fastcs/controllers/base_controller.py

```
@@ -168,47 +172,70 @@ def post_initialise(self):
+        """Validate all type-hints were introspected"""
         for name in self.__hinted_attributes:
             self._validate_hinted_attribute(name)
 
         for name in self.__hinted_sub_controllers:
             self._validate_hinted_controller(name)
 
+        for name in self.__hinted_methods:
+            self._validate_hinted_method(name)
+
         for subcontroller in self.sub_controllers.values():
             subcontroller._validate_type_hints()  # noqa: SLF001
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)